examples/music-player/src/3_HomePage.tsx

```diff
@@ -66,6 +71,8 @@ export function HomePage({ mediaPlayer }: { mediaPlayer: MediaPlayer }) {
     });
   };
 
+  const isAnonymousUser = useIsAnonymousUser();
```

Comment:
Will add a `useIsLocalOnly` to let devs change the logic based on the sync model.

It also kind-of explains better why share is disabled for anonymous users